modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/model/listener/APIEndpointRelevantObjectEntryModelListener.java

```diff
@@ -98,6 +98,31 @@ private boolean _isAPIApplication(long apiApplicationId) throws Exception {
 		return true;
 	}
 
+	private boolean _isAPISchema(long apiApplicationId) throws Exception {
+		if (apiApplicationId == 0) {
+			return true;
```

Comment:
Is this condition right? It seems it should be false  :thinking:

Reply:
I mean, the method is called `_isAPISchema`. I expect it only returns true if it is an API Schema; not if the id is 0 (that could be equivalent to null)